source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -325,8 +324,8 @@ export abstract class CardElement {
         return sizeChanged;
     }
 
-    isOnPage(): boolean {
-        return !!(this._renderedElement && this._renderedElement.offsetHeight);
+    isRendered(): boolean {
```

Comment:
Ah - now it looks like  this should be named isOverflowing, no?

Reply:
Still just a check that the card is in the DOM -- when `updateLayout()` gets called, we need to trigger the overflow handling logic regardless of whether or not the card actually does overflow, because it could be the case that there is now more space available for a previously truncated card.